packages/components/toasts/NormalToast.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import { TouchableOpacity, useWindowDimensions, View } from "react-native";
+
+import warningSVG from "../../../assets/icons/warning.svg";
+import {
+  errorColor,
+  neutral11,
+  neutral77,
+  successColor,
+} from "../../utils/style/colors";
+import { BrandText } from "../BrandText";
+import { SVG } from "../SVG";
+import { Box } from "../boxes/Box";
+import { SpacerRow } from "../spacer";
+
+const marginHorizontal = 24;
+
+export interface NormalToastProps {
+  title: string;
+  message?: string;
+  onPress?: () => void;
+  type?: "error" | "success";
+  topOffset?: number;
+}
+
+export const NormalToast: React.FC<NormalToastProps> = ({
```

Comment:
there is a regression in this component, on main it has an icon

<img width="449" alt="Screenshot 2024-02-28 at 12 20 47" src="https://github.com/TERITORI/teritori-dapp/assets/7917064/7e715533-d435-4571-b3d0-cc3b19cfa026">

on this pr the `!` icon does not appear